src/Superstructure/ESMFMod/include/ESMCI_Init.h

```diff
@@ -51,7 +52,9 @@ int ESMCI_Initialize(ESMC_CalKind_Flag defaultCalendar=ESMC_CALKIND_NOCALENDAR);
 int ESMCI_Initialize(int argc, char **argv,
   ESMC_CalKind_Flag defaultCalendar=ESMC_CALKIND_NOCALENDAR);
 
-int ESMCI_Finalize(void);
+int ESMCI_Finalize();
```

Comment:
Was this change from `ESMCI_Finalize(void)` to `ESMCI_Finalize()` intentional? I see that `ESMCI_Init.C` still defines it as `ESMCI_Finalize(void)`; should they be kept consistent?

Reply:
Purely an accident! I'll get that reverted, thanks for spotting that.